src/components/WwwFrame/TheHeader.vue

```diff
@@ -411,7 +413,7 @@ export default {
 		PromoBannerLarge,
 		PromoBannerSmall,
 		SearchBar,
-		TheLendMenu,
+		TheLendMenu: () => import('@/components/WwwFrame/LendMenu/TheLendMenu'),
```

Comment:
I added this back in, it was not the cause of the issues as I had initially thought